### ISSUE TITLE
Validation loss 

### ISSUE THREAD

**Issue by hannalovisao** [orrzohar/PROB#57]
Hi!
I am adapting the method to my own dataset and I want to make sure that the model does not overfit. How did you choose the number of epochs for the different tasks? Did you calculate the validation loss to see when it diverged from the training loss, or did you only look at the metrics such as current known mAP etc to see when they started to decline? 
BR, 
Hanna

**Comment by orrzohar** [comment]
Hi @hannalovisao,
I usually looked at the ratio between the known and unknown mAP and tried to find a good ratio between the two.
Usually, both known mAP and U-Recall go up in the beginning -  U-Recall faster than K-mAP. Then, K-mAP as K-mAP goes up U-Recall starts going down. 

<img width="941" alt="image" src="https://github.com/orrzohar/PROB/assets/108689663/7242b31f-a17a-44da-ac48-794e59ae5803">

Best,
Orr
